e2e/app/element/xpath-defaults.ts

```diff
@@ -27,9 +27,8 @@ export function labelXpath(opts: TextOptions) {
  * @param label
  */
 export function buttonXpath(opts: TextOptions) {
```

Comment:
I thought the linter was requiring return types. Anyway, it would be good to see those here.